app/controllers/application_controller.rb

```diff
@@ -1,5 +1,22 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  protect_from_forgery with: :null_session
+
+  def set_packages
+    packages = []
+    params[:packages].each do |package|
+      packages << ActiveShipping::Package.new(package[:weight].to_i, package[:dimensions].map(&:to_i))
+    end
+
+    return packages
+  end
+
+  def set_origin
+    ActiveShipping::Location.new(country: 'US', state: params[:origin][:state], city: params[:origin][:city], zip: params[:origin][:zip])
```

Comment:
It's a matter of what style guide you're using, but when the line is this long, I like to break it up into newlines. E.g.:
```
ActiveShipping::Location.new(
  country: 'US',
  state: params[:origin][:state],
[...]
```
This makes it more readable, but also makes on-the-job code reviews easier. Then if someone modifies a small part of just one of the elements, it's clear on the code review what they modified, rather than the whole line being colored.